CGMBLEKitUI/TransmitterSettingsViewController.swift

```diff
@@ -16,15 +17,29 @@ class TransmitterSettingsViewController: UITableViewController {
 
     let cgmManager: TransmitterManager & CGMManagerUI
 
-    private var glucoseUnit: HKUnit
+    private let displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
 
-    init(cgmManager: TransmitterManager & CGMManagerUI, glucoseUnit: HKUnit) {
+    private var cancellable: AnyCancellable?
```

Comment:
[nit] I prefer using `private lazy var cancellables = Set<AnyCancellable>()` and `.store(in: &cancellables)`, but that's just me.